climakitae/__init__.py

```diff
@@ -18,6 +18,7 @@
     # Methods
     "load",
     "export",
+    "remove_zarr",
```

Comment:
How come you add `remove_zarr` to the init? Is it so that you can get the `ck.remove_zarr` function call?

Reply:
Yes, that is where package level functions are referenced.